lib/depject/profile/sheet/edit.js

```diff
@@ -31,13 +32,15 @@ exports.create = function (api) {
       const currentName = api.about.obs.name(id)
       const currentImage = api.about.obs.image(id)
       const currentDescription = api.about.obs.description(id)
+      const currentPublicWebHosting = api.about.obs.latestValue(id, 'publicWebHosting')
```

Comment:
@reconbot Try using `api.about.obs.valueFrom(id, 'publicWebHosting', id)`

This is relevant also since using `valueFrom` ensures that it only shows self-updates (not updates made to feeds other than one's own).

Reply:
nice catch @clehner, that would have been awkward if anyone could set the property for anyone. 🙈

Reply:
@reconbot there's some debate about that. mostly because it bears the potential for abuse (assigning bad names to people) which would be... irrelevant, if it was just something everyone could set for themselves, but would be bad if the person's friends take that name over. I don't actually recall if there was any consensus on the matter...

But I'd regard that as a separate issue.

Reply:
From reading the source, we can see `ssb-about` uses https://github.com/ssbc/ssb-social-index. I don't actually know why latestValue wouldn't work; could be a bug in ssb-social-index.

For name, description, and image: maybe. Unlike publicWebHosting (in current known implementations), those properties may sometimes be set by others. Of course anyone can set any property for anyone (even if Patchwork doesn't have UI for it), but it's up to clients what they are going to consider valid or how they make use of the information. `about.socialValue` takes the social graph into account to compute `about` state. Patchwork makes use of that, including in this profile sheet page (via e.g. `api.about.obs.description`, defined in `lib/depject/about/obs.js`). For editing one's own profile, I think it would make sense to use what the user themself last set as the editor's initial state - so yes, using `valueFrom` here instead of `api.about.obs.name`, etc.. (Changing it more broadly should need more discussion). But this might not make much difference as I think socialValue prefers values set by the local user or by the target user - which would be the same as valueFrom in this case. It might only make a difference in the case where the user has not yet set their own value for the property.